### PR TITLE
Correct update and delete for dyn grouped labels

### LINKED ISSUE #341: Updating a dynamic grouped label returns error that the label does not exist
Rel #234

### PR DESCRIPTION
**Pull request by joshelser**
Closes #341

### CODE REVIEW

File: app/ui/labels.py [hunk 1]
```diff
@@ -257,7 +262,12 @@ def on_delete_click(self, name, is_dynamic):
             )
             with snowpark_session(self.snowflake) as txn:
                 # Make the old label, bypassing validation
-                label_to_del = ModelLabel.construct(name=name)
+                if is_dynamic:
+                    label_to_del = ModelLabel.construct(
+                        group_name=name, is_dynamic=True
```
Comment (vqmarkman) [inline app/ui/labels.py:267]: I'm confused :(   Isn't it supposed to be group_name=**group** here too ?

Comment (joshelser) [reply app/ui/labels.py:267]: The `name` variable here is either the `Label.name` or `Label.group_name`. It's just confusingly named.

https://github.com/sundeck-io/OpsCenter/pull/342/files/d4f1dc2d1c80c03e6a53c52b52aa732bb5c34c6f#diff-ca2f4a6b94e630f89baef30ac659472d5a12a08d453cdca6ae60e64ded9339d9L136-L148

Comment (vqmarkman) [reply app/ui/labels.py]: Thank you so much for making the change, my sanity is restored :)

**Review by vqmarkman — APPROVED** [review]

**Comment by vqmarkman** [comment]
The best part - Susannah has already merged a new test for dynamic labels.

**Pull request merged**